internal/types/plugin_policy.go

```diff
@@ -0,0 +1,74 @@
+package types
```

Comment:
rename back model and repo to policy.go

Reply:
[Done](https://github.com/vultisig/vultiserver-plugin/pull/90/commits/829b2755931333d0d970209c222f8ab9569b5b2b)